package/piwebcam/multi-gadget.sh

```diff
@@ -101,5 +103,4 @@ if [ "$CONFIGURE_USB_SERIAL" = true ] ; then
   config_usb_serial
 fi
 
-udevadm settle -t 5 || :
```

Comment:
Can this be removed?

Creating the usb-composite gagdets will create new devices in the kernel, and that will create udev events. It might be that the udev settle here is used to make sure we wait for those to exist before we move on ...

I don't think I've ever needed to do a udevadm settle here after creating the devices, but I probably normally ran things from the command line, so there could be races.

Reply:
I didn't consider that, there's no real reason /not/ to keep it there so probably worth adding back.